mobile/gen_bindings.sh

```diff
@@ -33,13 +33,13 @@ target_pkg="github.com/lightningnetwork/lnd/lnrpc"
 
 # A mapping from grpc service to name of the custom listeners. The grpc server
 # must be configured to listen on these.
-listeners="lightning=lightningLis walletunlocker=lightningLis state=lightningLis"
+listeners="lightning=lightningLis walletunlocker=lightningLis state=lightningLis autopilot=lightningLis chainnotifier=lightningLis invoices=lightningLis neutrinokit=lightningLis peers=lightningLis router=lightningLis signer=lightningLis walletkit=lightningLis watchtower=lightningLis watchtowerclient=lightningLis"
```

Comment:
We should also update the docs to explain what the new API object people will need to hit to access the extra sub-severs looks like.